### PR TITLE
Feature: Github action to build docker images and push to ghcr.io on release

### PR DESCRIPTION
**Pull request by Rkallenkoot**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Publish Docker image
+on:
+  release:
+    types: [published]
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
```
Comment (adamaltman) [inline .github/workflows/main.yml:7]: What is the corresponding GitHub pricing for this?

Comment (RomanHotsiy) [reply .github/workflows/main.yml:7]: > GitHub Packages is free for public repositories

Comment (Rkallenkoot) [reply .github/workflows/main.yml:7]: Like Roman said, public repo's are free.

**Review by Marshevskyy — APPROVED** [review]
looks good!

**Pull request merged**